indexers/vector/FaissIndexer/requirements.txt

```diff
@@ -1 +1 @@
-faiss-cpu
+faiss-cpu==1.6.3
```

Comment:
does `faiss-cpu==1.6.5` work as well?

Reply:
I do not know, I checked the current version and fixed the version that is currently installed, I can't go further and innovate fromt hat

Reply:
Pycharm was suggesting me `1.6.5` and now I tried it out. All tests pass.

Reply:
Done!